ps_contactinfo.php

```diff
@@ -71,7 +74,7 @@ public function renderWidget($hookName = null, array $configuration = [])
 
         if (preg_match('/^displayNav\d*$/', $hookName)) {
             $template_file = $this->templates['light'];
-        } elseif ($hookName == 'displayLeftColumn' || $hookName == 'displayRightColumn') {
+        } elseif ($hookName == 'displayContactLeftColumn' || $hookName == 'displayContactRightColumn' || $hookName == 'displayLeftColumn' || $hookName == 'displayRightColumn') {
```

Comment:
Can be replaced by [in_array](https://www.php.net/manual/en/function.in-array.php) to avoid multiple `||`

```
elseif (in_array($hookName, ['displayContactLeftColumn', 'displayContactRightColumn', 'displayLeftColumn', 'displayRightColumn']))
```